src/main/kotlin/com/sourcegraph/cody/statusbar/CodyStatusBarActionGroup.kt

```diff
@@ -20,14 +20,11 @@ class CodyStatusBarActionGroup : DefaultActionGroup() {
     e.presentation.isVisible = ConfigUtil.isCodyEnabled()
 
     removeAll()
-    if (e.project?.let { CodyStatusService.getCurrentStatus(it) } ==
-        CodyStatus.CodyAgentNotRunning) {
-      addAll(
-          OpenLogAction(),
-          AboutAction().apply { templatePresentation.text = "Open About To Troubleshoot Issue" },
```

Comment:
`AboutAction` here seems to be redundant. It makes the action group overpopulated. The GitHub report fills the about info automatically.